src/server/scripts/Events/midsummer.cpp

```diff
@@ -46,6 +46,19 @@ class go_midsummer_bonfire : public GameObjectScript
     }
 };
 
+enum torchToss
+{
+    GO_TORCH_TARGET_BRAZIER         = 187708,
+    GO_TORCH_TOSS_TARGET_BUNNY      = 25535,
```

Comment:
This is a creature though? Prefix it CREATURE

Reply:
Yes the bunny is NPC of course, will fix this